libres/lib/enkf/enkf_state.cpp

```diff
@@ -57,405 +53,411 @@
+
+            if (forward_load_context_accept_messages(load_context)) {
+                char *msg =
+                    util_alloc_sprintf("Failed to load: %s at step:%d",
+                                       enkf_node_get_key(node), report_step);
+                forward_load_context_add_message(load_context, msg);
+                free(msg);
+            }
+        }
+        enkf_node_free(node);
     }
-    enkf_node_free(node);
-  }
 }
 
+static void
+enkf_state_internalize_GEN_DATA(const ensemble_config_type *ens_config,
+                                forward_load_context_type *load_context,
+                                const model_config_type *model_config,
+                                int last_report) {
 
-static void enkf_state_internalize_GEN_DATA(const ensemble_config_type * ens_config,
-                                            forward_load_context_type * load_context ,
-                                            const model_config_type * model_config ,
-                                            int last_report) {
-
-  stringlist_type * keylist_GEN_DATA = ensemble_config_alloc_keylist_from_impl_type(ens_config, GEN_DATA);
+    stringlist_type *keylist_GEN_DATA =
+        ensemble_config_alloc_keylist_from_impl_type(ens_config, GEN_DATA);
 
-  int numkeys = stringlist_get_size(keylist_GEN_DATA);
+    int numkeys = stringlist_get_size(keylist_GEN_DATA);
 
-  if (numkeys > 0)
-    if (last_report <= 0)
-      res_log_fwarning("Trying to load GEN_DATA without properly "
-                       "set last_report (was %d) - will only look for step 0 data: %s",
-                       last_report,
-                       stringlist_iget(keylist_GEN_DATA, 0)
-        );
+    if (numkeys > 0)
+        if (last_report <= 0)
+            res_log_fwarning(
+                "Trying to load GEN_DATA without properly "
+                "set last_report (was %d) - will only look for step 0 data: %s",
+                last_report, stringlist_iget(keylist_GEN_DATA, 0));
 
-  const run_arg_type * run_arg = forward_load_context_get_run_arg(load_context);
-  enkf_fs_type * sim_fs        = run_arg_get_sim_fs(run_arg);
-  const int iens               = run_arg_get_iens(run_arg);
+    const run_arg_type *run_arg =
+        forward_load_context_get_run_arg(load_context);
+    enkf_fs_type *sim_fs = run_arg_get_sim_fs(run_arg);
+    const int iens = run_arg_get_iens(run_arg);
 
-  for (int ikey=0; ikey < numkeys; ikey++) {
-    const enkf_config_node_type * config_node = ensemble_config_get_node(ens_config,
-                                                                         stringlist_iget(keylist_GEN_DATA,
-                                                                                         ikey));
+    for (int ikey = 0; ikey < numkeys; ikey++) {
+        const enkf_config_node_type *config_node = ensemble_config_get_node(
+            ens_config, stringlist_iget(keylist_GEN_DATA, ikey));
 
-    /*
+        /*
```

Comment:
Indentation of comment now looks weird because the indentation of the code has changed.